docs/README.md

```diff
@@ -5,15 +5,17 @@ sidebar_position: 1
 
 # TigerBeetle Docs
 
-TigerBeetle is a distributed financial accounting database designed
-for mission critical safety and performance.
+TigerBeetle is a distributed financial accounting database designed for mission critical safety and
+performance.
 
-## Getting Started
+## Quick Start
 
-Head over to [Getting Started](./getting-started/README.md) to
-[download](./getting-started/download.md) TigerBeetle, [run a
-cluster](./getting-started/single-binary.md), and try [creating accounts and transfers using the
-CLI](./getting-started/cli-repl.md).
+[Download TigerBeetle](./quick-start/README.md) and run a
+[single-node](./quick-start/single-node.md) or [multi-node cluster](./quick-start/cluster.md) in
+under 2 minutes.
```

Comment:
:thinking: do we need an entire directory for a quick start? This is a start that is supposed to be quick, so constraining it to fit on a single page might be good forcing function for us to consider what is really important, and what should rather move to usage/guides/reference section.

Reply:
Good suggestion. I merged all the quick start pages into one and used tabs for the different flavors.

One difference from what you suggested is that I included the multi-node flavor alongside the single-node one, rather than it being a step afterwards. My rationale was that some people won't want to bother with that and it'll just repeat the same basic instructions with some slight tweaks that are more relevant for operators than application developers who want to get to using the API ASAP.